### PR TITLE
Fix Bug 1164216, add video to wom page and small other tweaks

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson @craigcook r?

### CODE REVIEW

**Comment by craigcook** [comment]
Since the video is in English I presume we should only show it to en-* locales. You can wrap the whole section in <code>{% if LANG.startswith('en') %}</code>.

Apart from that all looks good to me. r+wc :tomato:

**Comment by schalkneethling** [comment]
Thanks @craigcook !! Very good point, will inquire about whether or not we want to show the video only to en-*

**Comment by schalkneethling** [comment]
Asked https://bugzilla.mozilla.org/show_bug.cgi?id=1164216#c6

File: bedrock/mozorg/templates/mozorg/contribute/friends.html [hunk 1]
```diff
@@ -49,6 +49,16 @@ <h3 class="rewarded">{{_('Get rewarded')}}</h3>
...
 </section>
+
+{% if LANG.startswith('en') %}
+<hr />
+
+<section class="ff-video">
+  <h2>{{_('Watch the video')}}</h2>
+  <iframe src="https://www.youtube.com/embed/ZKcv8HHyyy8" frameborder="0" allowfullscreen></iframe>
+</section>
+{% endblock %}
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/contribute/friends.html]: oops, should be endif, not endblock

Comment (schalkneethling) [inline bedrock/mozorg/templates/mozorg/contribute/friends.html]: whoops, meh

**Pull request merged**